### PR TITLE
Fix close button size for price calculator

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
See title

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
* **#4888** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4888?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gustaveen and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/store/src/components/Header/HeaderMenu/HeaderMenu.css.ts [hunk 1]
```diff
@@ -29,15 +29,23 @@ export const backLink = style([
...
+  sprinkles({ display: 'flex', alignItems: 'center', justifyContent: 'center' }),
+  {
+    width: '2rem',
+    height: '2rem',
+    backgroundColor: tokens.colors.buttonSecondary,
+    borderRadius: tokens.radius.xxs,
+
+    ':hover': {
+      backgroundColor: tokens.colors.buttonSecondaryHover,
+    },
```
Comment (guilhermespopolin) [inline apps/store/src/components/Header/HeaderMenu/HeaderMenu.css.ts:42]: If it should always be a square we can leverage `aspect-ratio`:

```suggestion
export const backWrapper = style([
  sprinkles({ display: 'flex', alignItems: 'center', justifyContent: 'center' }),
  {
    width: '2rem',
    aspectRatio: '1 / 1',
    backgroundColor: tokens.colors.buttonSecondary,
    borderRadius: tokens.radius.xxs,

    ':hover': {
      backgroundColor: tokens.colors.buttonSecondaryHover,
    },
    
    ...responsiveStyles({
      lg: { width: '2.5rem' },
    }),
```

**Review by guilhermespopolin — APPROVED** [review]

File: apps/store/src/components/Header/HeaderMenu/HeaderMenu.css.ts (hunk 1 above)
Comment (gustaveen) [reply apps/store/src/components/Header/HeaderMenu/HeaderMenu.css.ts:42]: Yeah, I like that! 👌

**Comment by gustaveen** [comment]
### Merge activity

* **Oct 24, 5:14 AM EDT**: A user started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4888).

**Pull request merged**